projects/web-app/src/views/link.view.vue

```diff
@@ -23,7 +24,12 @@ import ExternalLink from '@shared/components/external-link.vue';
 export default {
   setup() {
     const headData = useHead().data;
-    return { headData };
+    const linkMetadata = computed(() => headData.value.head.link);
+
+    return {
+      headData,
+      linkMetadata,
```

Comment:
You can expose one of these two, but both are not needed.
I think you can also do `const { head } = useHead().data;` and then return `head` to the template, and in the template use `:data="head.link"`
Let me know if reactivity still works then, because I think that would be the cleanest solution generally.

Reply:
That will simplify things :) I will do this for the other views (pr's) as well.